### PR TITLE
Remove remote_execution_ssh_keys parameter step

### PR DESCRIPTION
**Pull request by ekohl**
This is done automatically by REX.

Includes https://github.com/theforeman/foreman-documentation/pull/2329, so currently a draft.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

**Review by adamlazik1 — APPROVED** [review]
Looks good stylewise. Also linked PR already merged.

File: guides/common/modules/proc_configuring-a-finish-template-for-amazon-ec2.adoc [hunk 1]
```diff
@@ -39,25 +39,15 @@ EOS
...
-In the corresponding *Value* field, enter `ec2-user`.
-ifdef::satellite,orcharhino[]
 .. In the *Name* field, enter `activation_keys`.
 In the corresponding *Value* field, enter your activation key.
 endif::[]
-ifdef::foreman-el,katello[]
-.. If you use the Katello plugin, in the *Name* field, enter `activation_keys`.
-In the corresponding *Value* field, enter your activation key.
+ifndef::katello,satellite,orcharhino[]
+. If you have the Remote Execution plugin installed, in *Host parameters*, click the *Add Parameter* button.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-a-finish-template-for-amazon-ec2.adoc:49]:
```suggestion
. If you have the Remote Execution plug-in installed, in *Host parameters*, click the *Add Parameter* button.
```

**Review by maximiliankolb — COMMENTED** [review]
One tiny suggestion, rest LGTM. Please rebase to master and resolve the merge conflict.

**Comment by ekohl** [comment]
Rebased

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   efa9c9421..9fc2e301d  3.8 -> 3.8
   9147cc034..450cfb20d  3.7 -> 3.7
   de791a58b..cfd281cc3  3.6 -> 3.6
   0ff301ee6..70e102f4d  3.5 -> 3.5
   73ab984a9..a1533d620  3.4 -> 3.4
   91ee05b91..df9da9649  3.3 -> 3.3

there are merge conflicts on 3.2 and 3.1. Please open a separate PR if you want this fixed. @ekohl

**Comment by ekohl** [comment]
I'm fine with leaving 3.2 & 3.1.

**Pull request merged**